### PR TITLE
Feature/review ios docs

### PR DESCRIPTION
**Pull request by RaimundasSakalauskas**
This PR introduces many changes to iOS section of the docs. iOS doc's haven't been updated in a while and these changes were long overdue.

### CODE REVIEW

File: src/content/reference/SDKs/ios.md [hunk 1]
```diff
@@ -858,10 +858,82 @@ List of function names exposed by device. Functions are only available after `re
...
+
+Mesh network state if device belongs to a mesh network. State can be pending if device is waiting for the role change confirmation.
+
+  `@property (nonatomic, readonly) BOOL isFlashing`
+
+Is device currently being flashed? Best effort - May not accurate reflect true state.
+
+  `@property (strong, nonatomic, nullable, readonly) NSString *lastIPAdress`
+
+Last know IP address of the device.
```
Comment (rickkas7) [inline src/content/reference/SDKs/ios.md]: Should be "last known"

File: src/content/reference/SDKs/ios.md [hunk 2]
```diff
@@ -858,10 +858,82 @@ List of function names exposed by device. Functions are only available after `re
...
+
+  `@property (strong, nonatomic, nullable, readonly) NSString *lastIccid`
+
+Last known ICCID (for cellular devices only)
+
+  `@property (strong, nonatomic, nullable, readonly) NSString *imei`
+
+Last know IMEI (for cellular devices only)
```
Comment (rickkas7) [inline src/content/reference/SDKs/ios.md]: "Last known"

**Review by rickkas7 — APPROVED** [review]
Looks good! Just two minor typos that I noticed.

File: src/content/reference/SDKs/ios.md (hunk 1 above)
Comment (RaimundasSakalauskas) [reply src/content/reference/SDKs/ios.md]: Later commits remove this section completely.

**Pull request merged**